complete/src/main/java/com/example/restservice/dao/Queue.java

```diff
@@ -17,24 +24,21 @@
   @GenericGenerator(name = "uuid2", strategy = "uuid2")
   private String queueId;
 
+  private boolean isPasswordProtected;
+  private int queuePassword;
```

Comment:
Do we really need two fields? Can we just use `private Integer queuePassword` and set it to `null` if the queue is not password protected? Again, open to discussion if you think a boolean field is required.

Reply:
I just followed whatever we had discussed in #30  . To have a boolean to say whether queue is password protected or not ,display password if it is present.

Reply:
oh I see. I think we can have this one Integer field.